package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "redux": "^4.2.0",
     "web-vitals": "^2.1.4"
```

Comment:
- [ ] Ensure you install the `redux toolkit` package. It should be present in this your package.json file.

Reply:
@DammyShittu , this is done